doc/contributions/build_install.md

```diff
@@ -0,0 +1,68 @@
+# Build & install
+
+The following page explains how to build and install the shadow-utils project.
```

Comment:
Just a small note that I think that mostly we refer to it as `shadow`. Afaik only the Fedora/RH distros call it `shadow-utils`